src/views/Simulation/utils/pixiUtils.js

```diff
@@ -11,92 +12,155 @@ const createGraphicsContext = (options) => {
     .stroke({ color: options.lineColor, width: 0.75 });
 };
 
-const createLinkGraphics = (links, setSelectedElement) => {
+const createLinkGraphics = (links, setSelectedElement, scale) => {
+  if (!links || !Array.isArray(links)) return [];
   return links.map((link) => {
     const graphics = new Graphics();
     graphics.alpha = 0.25;
     graphics.eventMode = 'static';
     graphics.cursor = 'pointer';
 
     const { source, target } = link;
-    graphics.moveTo(source.x + 4, source.y + 4);
+    graphics.moveTo((source.x + 4) * scale, (source.y + 4) * scale);
 
-    graphics.lineTo(target.x + 4, target.y + 4);
-    graphics.stroke({ width: 1, color: '#FFFFFF' });
+    graphics.lineTo((target.x + 4) * scale, (target.y + 4) * scale);
+    graphics.stroke({ width: 1 * scale, color: '#FFFFFF' });
```

Comment:
Not sure if this will work with the new graph layout and styles for nodes and edges :thinking:

Reply:
It has been changed during the rebase.